tests/api.c

```diff
@@ -18902,6 +18916,8 @@ static int test_wc_ed25519_import_private_key (void)
     const byte  pubKey[] = "Ed25519PublicKeyUnitTest......\n";
     word32      privKeySz = sizeof(privKey);
     word32      pubKeySz = sizeof(pubKey);
+    byte        bothKeys[sizeof(privKey) + sizeof(pubKey)];
```

Comment:
Won't these be unused variables if `#ifdef HAVE_ED25519_KEY_EXPORT` is not defined?

Reply:
done.   while I was at it, I fixed a couple other spots with broken handling for `HAVE_ED*_KEY_{EX,IM}PORT`:
```
 src/internal.c                 | 26 +++++++++++++++-------
 src/ssl.c                      | 60 ++++++++++++++++++++++++++++++++++----------------
 tests/api.c                    | 84 +++++++++++++++++++++++++++++++++++++---------------------------------
 tests/suites.c                 |  6 +++--
 wolfcrypt/src/asn.c            | 58 +++++++++++++++++++++++++-----------------------
 wolfssl/test.h                 | 16 +++++++-------
 wolfssl/wolfcrypt/asn_public.h | 17 ++++++++++++--
 wolfssl/wolfcrypt/ed25519.h    |  5 +++++
 wolfssl/wolfcrypt/ed448.h      |  6 +++++
 9 files changed, 172 insertions(+), 106 deletions(-)
```
:scream: